### PR TITLE
Login and Registration UX improvements

### PR DESCRIPTION
**Pull request by iheanyi**
Added in a logout functionality and also the persistence of the current user's session through sessionStorage.

### CODE REVIEW

File: assets/js/reducers/login.js [hunk 1]
```diff
@@ -36,4 +36,13 @@ function errors(state = {}, action) {
...
 }
 
-export default combineReducers({isLoggingIn, errors, showErrorMessages});
+function rememberSession(state = false, action) {
+  switch (action.type) {
+    case TOGGLE_REMEMBER_ME:
+      return !state;
+    default:
+      return state;
+  }
```
Comment (GRardB) [inline assets/js/reducers/login.js]: I think there are only ever two options for "Remember me," right? Perhaps we could just do:

    return (action.type === TOGGLE_REMEMBER_ME)
         ? !state
         : state;

Up to you, though. Consistency vs conciseness, I don't care.

**Comment by GRardB** [comment]
el jee em tea

File: assets/js/reducers/login.js (hunk 1 above)
Comment (reggie) [inline assets/js/reducers/login.js]: that syntax look uggs Gerald -_- but I agree, there's only two options I don't think a switch is necessary

Comment (iheanyi) [inline assets/js/reducers/login.js]: Good point y'all, I'll fix that up. Sorry, it's out of habit that I use a switch statement, my bad.

**Pull request merged**